content/influxdb3/enterprise/reference/cli/influxdb3/serve.md

```diff
@@ -115,17 +123,28 @@ You can use environment variables to define most `influxdb3 serve` options.
 For more information, see
 [Configuration options](/influxdb3/enterprise/reference/config-options/).
 
+## Key Requirements
+
+- **node-id**: Must be unique for any hosts sharing the same object store configuration
+- **cluster-id**: Must be unique and different from any node-id in your cluster
+- **object-store**: Determines where time series data is stored
+- **data-dir**: Specifies the path for local file storage, required with `--object-store file`
+
```

Comment:
These should be moved up to the other required options section. We should also mark the object store and data-dir options as required in the table.